addons/mail/controllers/discuss.py

```diff
@@ -138,15 +138,16 @@ def mail_message_post(self, thread_model, thread_id, post_data, **kwargs):
         return thread.message_post(**{key: value for key, value in post_data.items() if key in allowed_params}).message_format()[0]
 
     @http.route('/mail/message/update_content', methods=['POST'], type='json', auth='public')
-    def mail_message_update_content(self, message_id, body):
+    def mail_message_update_content(self, message_id, body, attachment_ids):
```

Comment:
Method should support body or attachment_ids not set to only update the other.
Doesn't this actually break delete?